src/audio/dai-zephyr.c

```diff
@@ -1203,7 +1203,8 @@ static int dai_copy(struct comp_dev *dev)
 	/* get data sizes from DMA */
 	ret = dma_get_status(dd->chan->dma->z_dev, dd->chan->index, &stat);
 	if (ret < 0) {
-		dai_report_xrun(dev, 0);
+		if (ret == -ENODATA)
+			dai_report_xrun(dev, 0);
```

Comment:
I recently found out that ALSA is using `EPIPE` to indicate xrun conditions, should we update our code to do the same?